jscomp/ml/typedecl.ml

```diff
@@ -433,21 +433,57 @@ let transl_declaration ~typeRecordAsObject env sdecl id =
                    | _ -> false) in
           let lbls_opt = match has_spread with
             | true ->
+              let substitute_type_vars type_vars typ =
+                let open Record_spread in
```

Comment:
skip this? Makes the code more readable with lsss magic

Reply:
Hmm, what do you mean?

Reply:
The `open`